### PR TITLE
Roles and Permissions for CTHS

### PR DESCRIPTION
**Pull request by mghughes**

### CODE REVIEW

File: cths_roles_and_permissions/cths_roles_and_permissions.info [hunk 1]
```diff
@@ -0,0 +1,96 @@
+name = CTHS Roles and Permissions
+core = 7.x
+package = Features
+dependencies[] = features
+dependencies[] = islandora
+dependencies[] = islandora_backup
```
Comment (jordandukart) [inline cths_roles_and_permissions/cths_roles_and_permissions.info]: Is the dependency on this and pathauto something that should be here? Not mentioned in the permissions spreadsheet and they are not in the environment as far as I can tell.

**Comment by jordandukart** [comment]
Seems like the submodule conflicts with platform_main_feature as well?

File: cths_roles_and_permissions/cths_roles_and_permissions.info (hunk 1 above)
Comment (mghughes) [reply cths_roles_and_permissions/cths_roles_and_permissions.info]: Removed - they were originally in the spec but taken out

**Comment by mghughes** [comment]
> Seems like the submodule conflicts with platform_main_feature as well?

This has been resolved as well - permissions were only listed in the `.info` of the main feature, so I've removed that.

**Comment by mghughes** [comment]
Resolved the merge conflict resulting from #250 getting merged

**Pull request merged**